Sources/GRPC/GRPCClientChannelHandler.swift

```diff
@@ -463,6 +463,8 @@ extension GRPCClientChannelHandler: ChannelInboundHandler {
       case let .decompressionLimitExceeded(compressedSize):
         return GRPCError.DecompressionLimitExceeded(compressedSize: compressedSize)
           .captureContext()
+      case let .lengthExceedsLimit(underlyingError):
+        return underlyingError.captureContext()
```

Comment:
When we create the error in `LengthPrefixedMessageReader.swift` we already capture the context. Is it right, that we capture the context here again, even though we pass the original error onwards?